### ISSUE TITLE
Examples have no language or direction

### ISSUE THREAD

**Issue by aphillips** [w3c/IndexedDB#180]
https://www.w3.org/TR/IndexedDB-2/#introduction

The examples contain natural language text, but no indication of language or text direction. This may be a distraction in the examples, but natural language has an influence on other operations (natural language sorting, etc.). Calling it out even though I don't expect a change.

This is I18N comment https://github.com/w3c/i18n-activity/issues/361. Please add the i18n-comment label.

**Comment by inexorabletash** [comment]
The strings in examples are a mix of identifiers and actual natural language text. If you have good examples of specs with code examples that do a good job of distinguishing these, and/or indicating language/text direction I'd love to take a peek.

Otherwise... probably a distraction. :(

**Comment by inexorabletash** [comment]
I went through all of the examples. Where natural language is used in strings (e.g. "Alice", "Bedrock Nights") I did not see anywhere that behavior would change if the strings were modified (direction, script, etc) other than where uniqueness was asserted. For example, none of the examples asserted that a particular ordering would be used. I realize that's not much of an assertion.

Also, the examples are always bare code - there's no surrounding HTML to mark up with language/direction. So... I don't think there's anything actionable here. But additional issues with concrete suggestions would be very welcome.

(As an aside, a general spec convention for "natural language" strings. vs identifiers in strings would be swell.)
